test/test-framework/pkg/apis/cache/v1alpha1/dummy_types.go

```diff
@@ -102,6 +107,16 @@ type Engine struct {
 	Pistons []string `json:"pistons"`
 }
 
+// +k8s:deepcopy-gen=false
+// +k8s:openapi-gen=false
+type Wheel struct {
+	// Type should be in spec with path equal to wheels[0].type
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Wheel Type"
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:arrayFieldGroup:wheels,urn:alm:descriptor:com.tectonic.ui:text"
+	Type string `json:"type"`
+}
+
 // +k8s:deepcopy-gen=false
 // +k8s:openapi-gen=false
 type InlinedComponent struct {
```

Comment:
Missing update the file gen with. 
Could you please run `./hack/generate/gen-test-framework.sh` to update the mock data as well?